balancer/frontend-v2
# Feat: Avalanche Support

# Description

Adds support for Avalanche

Related PR's: (Should all be merged first)

- https://github.com/balancer/balancer-sdk/pull/461
- https://github.com/balancer/tokenlists/pull/180
- https://github.com/balancer/balancer-api/pull/212/
- https://github.com/beethovenxfi/beethovenx-backend/pull/369

## Type of change


## How should this be tested?

- Select Avalanche network or go to /avalanche. 
- Ensure swaps / depositing / withdrawing works as expected. 

## Checklist:
